src/screens/HomeScreen/component.js

```diff
@@ -32,7 +31,7 @@ type Props = {
   featuredEventsTitle: string,
   featuredEvents: Event[],
   loading: boolean,
-  getAssetSource: FieldRef => ImageSource
+  getImageDetails: string => ?ImageDetails
```

Comment:
I had to keep this here in order to service the HeaderBanners need for dynamic sizing. See src/screens/HomeScreen/Header.js